community.md

```diff
@@ -48,33 +48,22 @@ We run an announcement-only, no-marketing-spam mailing list that we will use to
 <input class="cTextfieldstyle" maxlength="90" value="" id="emailUserOS" name="email" placeholder="I consent to join the email list" title="Email" type="text">
 </form> -->
 
-## Want to report issues? 
+## How you can help 
 
-Oh no, you hit a bump on the road? Please tell us - we want to fix all bugs!
+Getting a language established is like getting a new religion going. We really need you, our fans, to help spread the word.
 
-- Found a bug in the compiler, runtime, standard library, or tooling? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-lang/issues">ballerina-lang</a> repo.
-- Found a bug in the language specification? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-spec/issues">ballerina-spec</a> repo.
-- Found a bug on the website? Create an issue in the <a href="https://github.com/ballerina-platform/ballerina-dev-website/issues">ballerina-dev-website</a> repo. 
-- Found a security flaw? Send an email to security@ballerina.io. For details, see the <a href="https://ballerina.io/security/">security policy</a>.
+- Start with starring the <a href="https://github.com/ballerina-platform/ballerina-lang">main GitHub repo</a>. Thank you!
+- Follow us on Twitter: <a href="https://twitter.com/ballerinalang">@ballerinalang</a>. Tweet with the "#ballerinalang" hashtag.
+- Share the wealth by publishing your Ballerina module on <a href="https://central.ballerina.io">Ballerina Central</a>. 
+- Give us a buzz on <a href="mailto:contact@ballerina.io">contact@ballerina.io</a> if you want to organize a local meetup or hackathon. WSO2 will get right on it and help with presentation/training content, logistics, swag, and some funds for munchies.
+- Report an issue/bug in the below GitHub repos and tell us if you hit a bump on the road. We want to fix all bugs!

```

Comment:
What is the reason for merging the reporting issues section and helping us sections? IMO, reporting issues section should be highlighted first. That will give the idea of the basic flow - where we help the users first by resolving their issues first and then they help us by starring and tweeting after they are happy with the experience.

Reply:
How about something like the following for the complete flow? 
1. Invite the users to try out Ballerina (maybe we can point them to the quick start guide, BBEs, how-to guides and API docs) and develop something.
2. Ask them to create issues for any issues/bugs they find in the process.
3. Ask them to publish their work to Ballerina Central once they are ready so that others can benefit from their work.
4. Ask for their help if they liked the experience so far. e.g., starring the repo, following on Twitter, organizing local meetups etc.

Reply:
+1 for the flow @pubudu91 has suggested.

Reply:
Added the below sections:
- Try out Ballerina
- Seek for help
- Report issues
- Help us grow
- Contribute to Ballerina